libs/expo/betterangels/src/lib/screens/AddEditClient/AddEditClient.graphql

```diff
@@ -18,15 +18,22 @@ query GetClientProfile($id: ID!) {
   clientProfile(pk: $id) {
     ... on ClientProfileType {
       id
+      age
       address
       dateOfBirth
       gender
       hmisId
```

Comment:
please remove `hmisId` from all queries and mutations. `hmisId` should only be accessed via `hmisProfiles`, and not directly on/from clientprofile

Reply:
done